package-lock.json

```diff
@@ -1,3 +1,2761 @@
 {
-  "lockfileVersion": 2
+  "name": "wunderio-drupal-project",
```

Comment:
So far we've avoided adding lockfile dependencies to the repo for maintenance reasons. See https://github.com/wunderio/drupal-project/issues/143 & https://github.com/wunderio/drupal-project/pull/155. Should we change this?